### PR TITLE
Chore | Unify <Tabs /> component

### PR DESCRIPTION
**Pull request by Sharqiewicz**
Our current issue is the nesting components and logic within the /pages directory, they are not reusable at all.
In this PR, the `<Tabs>` component has been refactored and extracted from all instances within /pages to enhance reusability. It has been implemented in both the Spacewalk Bridge and Fund Wallet pages

### CODE REVIEW

**Comment by Sharqiewicz** [comment]
The commits are a result of having checked out this branch from the fund-wallet branch. Only the last five commits contain the changes for this PR

File: src/components/Tabs/Tab/index.tsx [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
-export function Tab<T>({ index, label, activeTab, setActiveTab }: TabItemProps<T>) {
+export function Tab<T>({ index, children, activeTab, setActiveTab, className, activeClassName }: TabProps<T>) {
   const getTabProps = (tabIndex: T) => ({
     'data-active': activeTab === tabIndex,
     onClick: () => setActiveTab(tabIndex),
   });
 
+  const defaultClassName = 'group tab relative w-full h-full p-2 ';
+
+  const combinedClassName = className || 'h-full text-lg font-bold text-primary py-5';
```
Comment (ebma) [inline src/components/Tabs/Tab/index.tsx:20]: I think this name is a bit misleading. I would expect it to be a combination of something while that's only done in l. 24. Maybe rename to `stylingClassName` or something?

File: src/components/Tabs/Tab/index.tsx [hunk 2]
```diff
@@ -1,33 +1,37 @@
...
-      {activeTab === index && (
-        <motion.div
-          layoutId="bubble"
-          // @ts-expect-error Caused by Preact, remove this comment once migrated to React
-          className="absolute inset-0 z-10 rounded-lg bg-primary"
-          transition={{ type: 'spring', bounce: 0.2, duration: 0.6 }}
-        />
-      )}
-    </a>
+    <>
```
Comment (ebma) [inline src/components/Tabs/Tab/index.tsx:23]: Do we need the outer `<></>`?

File: src/pages/fund-wallet/FundWalletTabs/index.tsx [hunk 3]
```diff
@@ -7,9 +7,9 @@ interface FundWalletTabsProps {
 }
 
 export function FundWalletTabs({ activeTab, setActiveTab }: FundWalletTabsProps) {
-  const tabItems = [
-    { index: FundWalletTab.Buy, label: 'Buy' },
-    { index: FundWalletTab.Exchange, label: 'Exchange' },
+  const tabItems: { index: FundWalletTab; children: JSX.Element }[] = [
```
Comment (ebma) [inline src/pages/fund-wallet/FundWalletTabs/index.tsx:10]: Nit: Maybe extract `{ index: FundWalletTab; children: JSX.Element }` to a new type to improve readability.

**Review by ebma — APPROVED** [review]
Looks good to me overall 👍  it's nice that this improves reusability and now the spacewalk tabs also have a smooth animation.

**Comment by Sharqiewicz** [comment]
@ebma Very good catches, applied all of them ✅

**Review by ebma — APPROVED** [review]

**Pull request merged**